dv/formal/rtl-changes.patch

```diff
@@ -1,46 +0,0 @@
-diff --git a/rtl/ibex_id_stage.sv b/rtl/ibex_id_stage.sv
-index 3a358af7..3c19d763 100644
---- a/rtl/ibex_id_stage.sv
```

Comment:
Hmm. Did we need this patch in the previous commit? If not, can we forget about it entirely?

Reply:
In the previous commit the README said to apply this patch manually. This file is necessary because it is now automatically applied by fusesoc.